package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "egeria-api-mocks",
+  "version": "0.0.0",
+  "description": "Egeria API Mocks",
+  "main": "index.js",
+  "scripts": {
+    "start": "node .",
+    "start:dev": "nodemon ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/egeria/egeria-api-mocks.git"
+  },
+  "keywords": [
```

Comment:
maybe add odpi and/or 'odpi egeria' . Perhaps 'metadata' ?

Reply:
> maybe add odpi and/or 'odpi egeria' . Perhaps 'metadata' ?

i think it's ok for now since this package will not be used in production..